be/test/storage/lake/vacuum_test.cpp

```diff
@@ -1634,4 +1635,180 @@ TEST(LakeVacuumTest2, test_delete_files_retry4) {
     EXPECT_GT(attempts, 1);
 }
 
+// NOLINTNEXTLINE
```

Comment:
curious what's wrong with the next line for clang format?

Reply:
Useless here, I think maybe I introduced it by mistake.... I can remove this in next change.